components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -456,50 +457,23 @@ open class PlacesReaderConnection internal constructor(connHandle: Long, conn: U
         }
     }
 
-    override fun getBookmark(guid: String): BookmarkTreeNode? {
-        readQueryCounters.measure {
-            val rustBuf = rustCall { err ->
-                LibPlacesFFI.INSTANCE.bookmarks_get_by_guid(this.handle.get(), guid, 0.toByte(), err)
-            }
-            try {
-                return rustBuf.asCodedInputStream()?.let { stream ->
-                    unpackProtobuf(MsgTypes.BookmarkNode.parseFrom(stream))
-                }
-            } finally {
-                LibPlacesFFI.INSTANCE.places_destroy_bytebuffer(rustBuf)
-            }
+    override fun getBookmark(guid: String): PublicNode? {
+        return readQueryCounters.measure {
+            this.conn.bookmarksGetByGuid(guid, 0.toByte().toUByte())
```

Comment:
I think we can pass `false` here (uniffi itself hides the strangeness with bools).